components/daterange/component.tsx

```diff
@@ -1,47 +1,58 @@
 import React from 'react';
 import DatePicker from '../datepicker';
 import addDays from 'date-fns/addDays';
+import useChangeEffect from 'components/hooks/useChangeState';
 
 interface IDateRangePicker {
   startDate?: Date;
   endDate?: Date;
   startPlaceHolder?: string;
   endPlaceHolder?: string;
+  onChange?: (startDate?: Date, endDate?: Date) => void;
 }
 
 const DateRangePicker: React.FC<IDateRangePicker> = ({
   startDate,
   endDate,
   startPlaceHolder,
   endPlaceHolder,
+  onChange,
 }) => {
   const [sStartDate, setStartDate] = React.useState(startDate);
   const [sEndDate, setEndDate] = React.useState(endDate);
 
+  useChangeEffect(() => {
+    onChange(sStartDate, sEndDate);
```

Comment:
<img width="529" alt="image" src="https://user-images.githubusercontent.com/9701591/167620515-281dc641-c136-4655-9cbc-09e9d0127fa4.png">
The datepicker is not working correctly right now. Check the image